tutorial.md

```diff
@@ -20,8 +20,8 @@ Before starting the tutorial, be sure to checkout the `tutorial` branch. We also
 
 ```bash
 $ git clone https://github.com/watson-developer-cloud/simple-chat-swift.git
-$ git checkout tutorial
 $ cd simple-chat-swift/simple-chat
+$ git checkout tutorial
 $ git submodule init
 $ git submodule update
 $ carthage update --platform iOS
```

Comment:
Might want `$ pod install` here.

Reply:
I had forgotten, but it turns out that the convention is to check-in the `Pods` directory. I had done that previously, and updated the dependencies with this pull request.

(However, by convention, the `Carthage` directory is not checked-in, so users should do a `carthage update`.)